packages/repository-modernizer/src/restructure-filters.js

```diff
@@ -161,7 +197,11 @@ function writeFilterPathsToFilterXml(filterPaths, filePath, conversionStep) {
 }
 
 function isImmutableContentFilter(line) {
+    if(line.indexOf('"')== -1){
+        line = line.substring(line.indexOf('<root>')+6);
```

Comment:
Please use constant. Also `6` value should be replaced with String.length functionality.